web/themes/new_weather_theme/templates/partials/hourly-table.html.twig

```diff
@@ -1,7 +1,13 @@
-{% set times = hours | map(h => h.time) | json_encode %}
-{% set temps = hours | map(h => h.temperature) | json_encode %}
-{% set feelsLike = hours | map(h => h.apparentTemperature) | json_encode %}
-{% set pops = hours | map(h => h.probabilityOfPrecipitation) | json_encode %}
+{#
+  Available variables:
+    hours - An array of all the hours that should be represented in the table.
+    alerts - An array of all the alerts that intersect with those hours.
+#}
+
+{% set times = hours | map(h => h.hour) | json_encode %}
+{% set temps = hours | map(h => h.temperature.degF) | json_encode %}
```

Comment:
We can't toggle F/C yet?

Reply:
Correct. There's an in-process design ticket for it, and we opted to postpone the engineering work until after the interop layer goes in because it transparently provides data in all the units we expect at once, so swapping between them can be completely client-side.